src/mmrelay/db_utils.py

```
@@ -519,39 +817,141 @@ def wipe_message_map():
+                cursor,
+                meshtastic_id,
+                matrix_event_id,
+                matrix_room_id,
+                meshtastic_text,
+                meshtastic_meshnet,
+            ),
+            write=True,
+        )
+    except sqlite3.Error as e:
+        logger.exception(
+            f"Database error storing message map for {matrix_event_id}: {e}"
```

Ruff (0.14.3)

Redundant exception object included in `logging.exception` call

(TRY401)